### PR TITLE
GIX-1688: Clean up after new neuron details page

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Clean up unused components after new neuron details page.

# Changes

* Remove NnsNeuronMetaInfoCard component, spec file and PO.
* Remove NnsNeuronInfoStake component, spec file and PO.
* Remove NnsNeuronMaturityCard component, spec file and PO.
* Remove NeuronJoinFundCard component, spec file and PO.
* Remove SnsNeuronMetaInfoCard component, spec file and PO.
* Remove SnsNeuronInfoStake component, spec file and PO.
* Remove SnsNeuronMaturityCard component, spec file and PO.
* Remove expectation that old components are not rendered in Nns and Sns neuron detail page spec file.
* Remove PO methods related to remove components.

# Tests

Mentioned above.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/tests/page-objects/NnsNeuronDetail.page-object.ts [hunk 1]
```diff
@@ -71,32 +56,12 @@ export class NnsNeuronDetailPo extends BasePageObject {
...
-
-    return this.getPageHeaderPo().isPresent();
-  }
-
   async getNeuronId(): Promise<string> {
-    if (await this.isNewUi()) {
-      return this.getNeuronIdNewUi();
-    }
-    return this.getNeuronIdOldUi();
+    return this.getNeuronIdNewUi();
```
Comment (dskloetd) [inline frontend/src/tests/page-objects/NnsNeuronDetail.page-object.ts:64]: Just do
```
return this.getPageHeaderPo().getNeuronId();
```
here and remove `getNeuronIdNewUi`?

Comment (lmuntaner) [reply frontend/src/tests/page-objects/NnsNeuronDetail.page-object.ts:64]: I'm planning to clean that in another PR.

Comment (dskloetd) [reply frontend/src/tests/page-objects/NnsNeuronDetail.page-object.ts:64]: In that case please keep a TODO referencing GIX-1688.

**Review by dskloetd — APPROVED** [review]

File: frontend/src/tests/page-objects/NnsNeuronDetail.page-object.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/page-objects/NnsNeuronDetail.page-object.ts:64]: I have it in my own todo list. No worries.

**Pull request merged**